openlibrary/plugins/openlibrary/js/graphs/plot.js

```diff
@@ -102,9 +102,9 @@ export function loadEditionsGraph() {
     }
 
     if (dateFrom == (dateTo - 1)) {
-        $('.clickdata').text(`$_('published in') ${dateFrom}`);
+        $('.clickdata').text(`Published in ${dateFrom}`);
```

Comment:
This has nothing to do with the plotting/graphing functionality and should be moved to the page template

Reply:
@tfmorris all JS is being moved to the openlibrary/plugins/openlibrary/js folder so please don't recommend putting any JS back in templates to avoid confusion.

If this has nothing to do with the graphs it should be moved somewhere else e.g. the main entry point index.js but certainly not back into a template.

I am a little confused why yu think this has nothing to do with graphs though - this code runs inside a plothover event - `placeholder.bind('plothover', function (event, pos, item) {`  - that seems pretty graphy to me!

Reply:
I think it needs to be in the JS because it changes based on the brush graph and what's selected. (or at least it's supposed to, looking at the code). Closing as won't fix.